.vitepress/theme/components/Feedback/Feedback.vue

```diff
@@ -85,7 +85,7 @@ watch(
 
 <template>
   <div class="feedbackContainer">
-    <form name="webcontainer-api-doc-feedback" data-netlify="true" hidden>
+    <form name="stackblitz-doc-feedback" data-netlify="true" hidden>
```

Comment:
Can we extract the form name to a constant, and then use it in both places?
Avoiding the duplication where we hardcode it in two places should avoid repeating this issue in the future.

For instance it can be a constant in the setup script:

```ts
const netlifyFormName = 'stackblitz-doc-feedback';
```

or it could be defined as a prop received by the feedback component, and then defined at the call site:

```vue
<Feedback netlifyFormName="stackblitz-doc-feedback" />
```